frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -251,7 +251,8 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                     sources.forEach((s) => {
                         const k = keyForSource(s)
                         // we just need something against each key so we don't load it again
-                        cache.snapshotsBySource[k] = cache.snapshotsBySource[k] || { snapshots: [] }
+                        cache.snapshotsBySource[k] = cache.snapshotsBySource[k] || {}
```

Comment:
previously we were dumping info here to infer that it had been loaded
then we started adding other info
so the inferring of status broke
now we have an explicit marker